ui/src/utils/helpText.js

```diff
@@ -0,0 +1,27 @@
+const helpText = {
+  namespace:
+    'Logical grouping of rules. This is often times your service name.',
```

Comment:
Perhaps change "This is often times your service name." to "The namespace name must be a valid service name"?

Reply:
Done